android/app/build.gradle

```diff
@@ -29,7 +29,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion flutter.compileSdkVersion
```

Comment:
https://stackoverflow.com/questions/53977807/flutter-compilesdkversion-and-targetsdkversion-on-android

ควรแก้ minSdkVersion?